python/examples/advanced/Image_Logging.ipynb

```diff
@@ -0,0 +1,282 @@
+          ]
+        }
+      ],
+      "source": [
+        "results = log_image({\"red\": img1, \"blue\": img2})\n",
```

Comment:
I find the choice of name here of "red" to be potentially confusing since you might want to split color channels but this is not what the example is doing, better would be to name it something indicating the source of the image, e.g. we could show that you can log a camera rig or position of object kind of named inputs, e.g.:
`log_image({"front_camera": img1, "back_camera": img2})`